CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ include (CMakeLists_files.cmake)
 macro (config_hook)
   opm_need_version_of ("dune-common")
   opm_need_version_of ("dune-istl")
+  opm_need_version_of ("dune-fem")
```

Comment:
needs the appropriate if(), this is not a hard dependency.
```cmake
if(dune-fem_FOUND)
endif()
```

Reply:
as you see from the passed build, it's not strictly a necessity. but if i saw that without the if i would be very confused.

Reply:
Sure. Clear enough for me.